frontend/app/services/MemberService.scala

```diff
@@ -305,7 +305,7 @@ class MemberService(identityService: IdentityService,
 
   override def getUsageCountWithinTerm(subscription: Subscription, unitOfMeasure: String): Future[Option[Int]] = {
     val features = subscription.features
-    val startDate = subscription.startDate.toDateTimeAtCurrentTime()
+    val startDate = subscription.startDate.toDateTimeAtStartOfDay(DateTimeZone.forID("America/Los_Angeles"))
```

Comment:
gosh, yes, not my finest hour 😨  I think I might've had rebase troubles or something.